2018-09-13 10:35:13.039 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:35:13.044 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:36:13.038 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:36:13.042 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:37:13.039 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:37:13.042 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:38:04.915 4778 DEBUG oslo_concurrency.lockutils [req-e44a8b87-2889-40ec-9150-1bbf1b69486a 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:05.680 4778 DEBUG neutron.agent.dhcp.agent [req-a683f709-d64a-4fc8-ad47-bb71c9916e6e - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:05.682 4778 DEBUG neutron.agent.linux.utils [req-a683f709-d64a-4fc8-ad47-bb71c9916e6e - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:05.683 4778 DEBUG neutron.agent.linux.dhcp [req-a683f709-d64a-4fc8-ad47-bb71c9916e6e - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525 on network daa3e87e-298e-4b6d-813d-5be8d62b3525 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-13 10:38:05.683 4778 DEBUG neutron.agent.linux.dhcp [req-a683f709-d64a-4fc8-ad47-bb71c9916e6e - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525 on network daa3e87e-298e-4b6d-813d-5be8d62b3525 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-13 10:38:08.883 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:08.907 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:08.932 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:08.958 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:08.973 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:08.997 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap1acd1762-9e', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.031 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-o', 'link', 'show', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.052 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-13 10:38:09.078 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap1acd1762-9e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:38:09.079 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-13 10:38:09.081 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap1acd1762-9e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:38:09.083 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap1acd1762-9e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:38:09.084 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'1acd1762-9e3f-4652-8a45-e4b14c827c87', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:cd:dc:ab'})), record=tap1acd1762-9e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:38:09.110 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap1acd1762-9e', 'address', 'fa:16:3e:cd:dc:ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.127 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.139 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap1acd1762-9e', 'netns', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.164 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap1acd1762-9e', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.178 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap1acd1762-9e', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.198 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Acquired semaphore "iptables-qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:09.200 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.227 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.253 4778 DEBUG neutron.agent.linux.iptables_manager [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-13 10:38:09.258 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Releasing semaphore "iptables-qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:09.261 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-o', 'link', 'show', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.283 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'addr', 'show', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.303 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'addr', 'add', '11.141.1.2/24', 'scope', 'global', 'dev', 'tap1acd1762-9e', 'brd', '11.141.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.326 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.348 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Setting IPv4 gateway for dhcp netns on net daa3e87e-298e-4b6d-813d-5be8d62b3525 to 11.141.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-13 10:38:09.350 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.141.1.1', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.366 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.387 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Cleaning stale devices for network daa3e87e-298e-4b6d-813d-5be8d62b3525 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-13 10:38:09.389 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.404 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.424 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Building initial lease file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-13 10:38:09.426 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Done building initial lease file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases with contents:
1536935889 fa:16:3e:cd:dc:ab 11.141.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-13 10:38:09.427 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:09.430 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:09.434 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:09.436 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host', '--addn-hosts=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap1acd1762-9e', '--dhcp-range=set:tag0,11.141.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:09.479 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:09.481 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:09.483 4778 DEBUG neutron.agent.linux.external_process [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:09.485 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:09.503 4778 DEBUG oslo_concurrency.lockutils [req-e44a8b87-2889-40ec-9150-1bbf1b69486a 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:10.177 4778 DEBUG neutron.agent.dhcp.agent [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:10.179 4778 DEBUG neutron.agent.linux.dhcp [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:10.183 4778 DEBUG neutron.agent.linux.dhcp [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:10.191 4778 DEBUG neutron.agent.linux.utils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.214 4778 DEBUG neutron.agent.linux.dhcp [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:10.216 4778 DEBUG neutron.agent.linux.utils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.237 4778 DEBUG neutron.agent.linux.utils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.257 4778 DEBUG neutron.agent.linux.utils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:10.258 4778 DEBUG neutron.agent.linux.utils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:10.259 4778 DEBUG neutron.agent.linux.external_process [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:10.260 4778 DEBUG oslo_concurrency.lockutils [req-63c82838-8063-45d0-a42a-d5365007cddc - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:10.261 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:10.262 4778 DEBUG neutron.agent.dhcp.agent [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:38:06Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.2'}], id=1acd1762-9e3f-4652-8a45-e4b14c827c87, mac_address=fa:16:3e:cd:dc:ab, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:06Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-13 10:38:10.263 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:10.264 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:10.265 4778 INFO neutron.agent.dhcp.agent [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-13T14:38:06Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.2'}], id=1acd1762-9e3f-4652-8a45-e4b14c827c87, mac_address=fa:16:3e:cd:dc:ab, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:08Z
2018-09-13 10:38:10.267 4778 DEBUG neutron.agent.dhcp.agent [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:10.268 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:10.270 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:10.275 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.296 4778 DEBUG neutron.agent.linux.dhcp [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:10.298 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.318 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:10.337 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:10.339 4778 DEBUG neutron.agent.linux.utils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:10.341 4778 DEBUG neutron.agent.linux.external_process [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:10.343 4778 DEBUG oslo_concurrency.lockutils [req-a3c0edf2-b8d4-4c4b-b76a-ac282a05288e - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:13.040 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:38:13.045 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:38:53.903 4778 DEBUG oslo_concurrency.lockutils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:53.905 4778 INFO neutron.agent.dhcp.agent [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:38:52Z, description=, device_id=deee62a3-df21-4012-8787-c96efbf57305, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.1'}], id=e49ac6e7-ac7f-4b8a-a14f-288100adb42e, mac_address=fa:16:3e:98:f5:29, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:52Z
2018-09-13 10:38:53.907 4778 DEBUG neutron.agent.dhcp.agent [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:53.909 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:53.914 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:53.920 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:53.950 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:53.951 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:53.984 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:54.002 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:54.005 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:54.006 4778 DEBUG neutron.agent.linux.external_process [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:54.010 4778 DEBUG oslo_concurrency.lockutils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:55.438 4778 DEBUG oslo_concurrency.lockutils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:55.441 4778 INFO neutron.agent.dhcp.agent [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:38:52Z, description=, device_id=deee62a3-df21-4012-8787-c96efbf57305, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.1'}], id=e49ac6e7-ac7f-4b8a-a14f-288100adb42e, mac_address=fa:16:3e:98:f5:29, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:52Z
2018-09-13 10:38:55.442 4778 DEBUG neutron.agent.dhcp.agent [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:55.444 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:55.448 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:55.455 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:55.470 4778 DEBUG neutron.agent.linux.dhcp [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:55.472 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:55.489 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:55.509 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:55.510 4778 DEBUG neutron.agent.linux.utils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:55.511 4778 DEBUG neutron.agent.linux.external_process [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:55.513 4778 DEBUG oslo_concurrency.lockutils [req-e074a7ae-7a0e-4367-8366-b2bc6f7a7ba6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:58.772 4778 DEBUG oslo_concurrency.lockutils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:58.774 4778 INFO neutron.agent.dhcp.agent [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:38:52Z, description=, device_id=deee62a3-df21-4012-8787-c96efbf57305, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.1'}], id=e49ac6e7-ac7f-4b8a-a14f-288100adb42e, mac_address=fa:16:3e:98:f5:29, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:54Z
2018-09-13 10:38:58.776 4778 DEBUG neutron.agent.dhcp.agent [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:58.779 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:58.783 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:58.790 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:58.807 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:58.809 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:58.829 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:58.847 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:58.849 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:58.850 4778 DEBUG neutron.agent.linux.external_process [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:58.851 4778 DEBUG oslo_concurrency.lockutils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:38:59.658 4778 DEBUG oslo_concurrency.lockutils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:38:59.659 4778 INFO neutron.agent.dhcp.agent [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-13T14:38:52Z, description=, device_id=deee62a3-df21-4012-8787-c96efbf57305, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.1'}], id=e49ac6e7-ac7f-4b8a-a14f-288100adb42e, mac_address=fa:16:3e:98:f5:29, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:38:59Z
2018-09-13 10:38:59.660 4778 DEBUG neutron.agent.dhcp.agent [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:38:59.662 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:38:59.664 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:38:59.668 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:59.686 4778 DEBUG neutron.agent.linux.dhcp [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:38:59.688 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:59.709 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:38:59.733 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:59.734 4778 DEBUG neutron.agent.linux.utils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:38:59.735 4778 DEBUG neutron.agent.linux.external_process [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:38:59.736 4778 DEBUG oslo_concurrency.lockutils [req-7f2ce6de-d6ce-4c56-b0e5-9d6f9ae8b91c - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:08.883 4778 DEBUG oslo_concurrency.lockutils [req-b028be88-bf68-4231-aab4-6a4ccea120a6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:08.884 4778 DEBUG oslo_concurrency.lockutils [req-b028be88-bf68-4231-aab4-6a4ccea120a6 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Releasing semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:13.039 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:39:13.044 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:39:13.326 4778 DEBUG oslo_concurrency.lockutils [req-16228fbb-86ef-4234-945f-16a111437656 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:13.327 4778 DEBUG oslo_concurrency.lockutils [req-16228fbb-86ef-4234-945f-16a111437656 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:14.196 4778 DEBUG oslo_concurrency.lockutils [req-16228fbb-86ef-4234-945f-16a111437656 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:14.197 4778 DEBUG oslo_concurrency.lockutils [req-16228fbb-86ef-4234-945f-16a111437656 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:35.870 4778 DEBUG oslo_concurrency.lockutils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:35.872 4778 INFO neutron.agent.dhcp.agent [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:39:34Z, description=, device_id=ae34b23d-63a9-416d-9701-225bd26b289b, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.9'}], id=6ec4f511-fe8d-463b-ac03-24efa0a6d7e2, mac_address=fa:16:3e:6d:ef:c6, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=True, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=3, security_groups=[u'd0c8d6c2-9d13-45ca-907d-4c8d83f0266e'], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:39:35Z
2018-09-13 10:39:35.874 4778 DEBUG neutron.agent.dhcp.agent [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:39:35.876 4778 DEBUG neutron.agent.linux.dhcp [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:39:35.934 4778 DEBUG neutron.agent.linux.dhcp [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:39:35.957 4778 DEBUG neutron.agent.linux.utils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:35.981 4778 DEBUG neutron.agent.linux.dhcp [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:39:35.983 4778 DEBUG neutron.agent.linux.utils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:36.003 4778 DEBUG neutron.agent.linux.utils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:36.016 4778 DEBUG neutron.agent.linux.utils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:36.018 4778 DEBUG neutron.agent.linux.utils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:36.019 4778 DEBUG neutron.agent.linux.external_process [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:39:36.020 4778 DEBUG oslo_concurrency.lockutils [req-bdc3cf54-7bed-47ea-a22d-6b5752b0e1a5 7e3296160ca0403d931c46d721211018 d924abce208c4c66b8ca02355a868a67 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:38.642 4778 DEBUG oslo_concurrency.lockutils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:38.643 4778 INFO neutron.agent.dhcp.agent [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:39:34Z, description=, device_id=ae34b23d-63a9-416d-9701-225bd26b289b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.9'}], id=6ec4f511-fe8d-463b-ac03-24efa0a6d7e2, mac_address=fa:16:3e:6d:ef:c6, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=True, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=4, security_groups=[u'd0c8d6c2-9d13-45ca-907d-4c8d83f0266e'], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:39:37Z
2018-09-13 10:39:38.645 4778 DEBUG neutron.agent.dhcp.agent [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:39:38.647 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:39:38.649 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:39:38.654 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:38.676 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:39:38.678 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:38.694 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:38.710 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:38.712 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:38.714 4778 DEBUG neutron.agent.linux.external_process [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:39:38.715 4778 DEBUG oslo_concurrency.lockutils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:39:39.335 4778 DEBUG oslo_concurrency.lockutils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:39:39.337 4778 INFO neutron.agent.dhcp.agent [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-13T14:39:34Z, description=, device_id=ae34b23d-63a9-416d-9701-225bd26b289b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.9'}], id=6ec4f511-fe8d-463b-ac03-24efa0a6d7e2, mac_address=fa:16:3e:6d:ef:c6, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=True, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=5, security_groups=[u'd0c8d6c2-9d13-45ca-907d-4c8d83f0266e'], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:39:38Z
2018-09-13 10:39:39.339 4778 DEBUG neutron.agent.dhcp.agent [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:39:39.342 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:39:39.346 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:39:39.351 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:39.369 4778 DEBUG neutron.agent.linux.dhcp [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:39:39.372 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:39.392 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:39:39.415 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:39.417 4778 DEBUG neutron.agent.linux.utils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:39:39.418 4778 DEBUG neutron.agent.linux.external_process [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:39:39.421 4778 DEBUG oslo_concurrency.lockutils [req-389f38b7-63b6-46a7-a451-37f4d68a5ef2 23e04251e35a4af8836d799cdbc6c009 4eeeddb45ad446a68ed4de7bd550932a - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:40:06.821 4778 DEBUG oslo_concurrency.lockutils [req-e2b45162-62a1-466f-a3f6-7d8ee4a0272d 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:40:06.822 4778 DEBUG oslo_concurrency.lockutils [req-e2b45162-62a1-466f-a3f6-7d8ee4a0272d 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Releasing semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:40:07.872 4778 DEBUG oslo_concurrency.lockutils [req-e2b45162-62a1-466f-a3f6-7d8ee4a0272d 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:40:07.872 4778 DEBUG oslo_concurrency.lockutils [req-e2b45162-62a1-466f-a3f6-7d8ee4a0272d 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Releasing semaphore "dhcp-agent-network-lock-bcf73fc2-8df6-4f12-801c-ef4abf269c73" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:40:13.040 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:40:13.044 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:41:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:41:13.046 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:42:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:42:13.043 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:43:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:43:13.045 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:44:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:44:13.048 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:44:40.203 4778 DEBUG oslo_concurrency.lockutils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:44:40.207 4778 DEBUG neutron.agent.dhcp.agent [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:44:40.209 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'dhcp_release', 'tap1acd1762-9e', '11.141.1.9', 'fa:16:3e:6d:ef:c6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:44:40.237 4778 DEBUG neutron.agent.linux.dhcp [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:44:40.249 4778 DEBUG neutron.agent.linux.dhcp [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:44:40.259 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:44:40.278 4778 DEBUG neutron.agent.linux.dhcp [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:44:40.281 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:44:40.299 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:44:40.321 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:44:40.324 4778 DEBUG neutron.agent.linux.utils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:44:40.325 4778 DEBUG neutron.agent.linux.external_process [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:44:40.327 4778 DEBUG oslo_concurrency.lockutils [req-bfdbd18a-d3a4-4a36-926e-cc3c5643b571 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:45:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:45:13.048 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:45:40.758 4778 DEBUG oslo_concurrency.lockutils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:45:40.760 4778 DEBUG neutron.agent.dhcp.agent [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:45:40.763 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'dhcp_release', 'tap1acd1762-9e', '11.141.1.1', 'fa:16:3e:98:f5:29'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:45:40.790 4778 DEBUG neutron.agent.linux.dhcp [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:45:40.795 4778 DEBUG neutron.agent.linux.dhcp [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:45:40.803 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:45:40.815 4778 DEBUG neutron.agent.linux.dhcp [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:45:40.818 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:45:40.832 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap1acd1762-9e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:45:40.848 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:45:40.849 4778 DEBUG neutron.agent.linux.utils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:45:40.850 4778 DEBUG neutron.agent.linux.external_process [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:45:40.851 4778 DEBUG oslo_concurrency.lockutils [req-76255c22-4e58-4975-9a3a-b91758298dbc 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:13.041 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:46:13.048 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-13 10:46:13.852 4778 DEBUG oslo_concurrency.lockutils [req-e67ebc3c-5ba2-4a9c-b61c-e463c4e88c4b 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:13.854 4778 DEBUG neutron.agent.dhcp.agent [req-e67ebc3c-5ba2-4a9c-b61c-e463c4e88c4b 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:13.859 4778 DEBUG neutron.agent.linux.utils [req-e67ebc3c-5ba2-4a9c-b61c-e463c4e88c4b 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['kill', '-9', '12249'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:13.886 4778 DEBUG neutron.agent.linux.utils [req-e67ebc3c-5ba2-4a9c-b61c-e463c4e88c4b 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-13 10:46:13.962 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap1acd1762-9e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:14.044 4778 DEBUG neutron.agent.linux.utils [req-f09b38fa-2414-4e69-a800-6444d73f3e41 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:14.059 4778 DEBUG neutron.agent.linux.utils [req-f09b38fa-2414-4e69-a800-6444d73f3e41 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:14.074 4778 DEBUG oslo_concurrency.lockutils [req-f09b38fa-2414-4e69-a800-6444d73f3e41 - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:17.476 4778 DEBUG neutron.agent.dhcp.agent [req-feca998f-c1a4-468d-b58b-195ed7380797 - - - - -] resync (daa3e87e-298e-4b6d-813d-5be8d62b3525): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-13 10:46:17.477 4778 INFO neutron.agent.dhcp.agent [req-feca998f-c1a4-468d-b58b-195ed7380797 - - - - -] Synchronizing state
2018-09-13 10:46:18.270 4778 INFO neutron.agent.dhcp.agent [req-8a2cf9fc-f5c9-45c2-9719-e9d9e3ca0877 - - - - -] All active networks have been fetched through RPC.
2018-09-13 10:46:18.273 4778 INFO neutron.agent.dhcp.agent [-] Starting network bcf73fc2-8df6-4f12-801c-ef4abf269c73 dhcp configuration
2018-09-13 10:46:18.274 4778 INFO neutron.agent.dhcp.agent [-] Finished network bcf73fc2-8df6-4f12-801c-ef4abf269c73 dhcp configuration
2018-09-13 10:46:18.276 4778 INFO neutron.agent.dhcp.agent [-] Starting network daa3e87e-298e-4b6d-813d-5be8d62b3525 dhcp configuration
2018-09-13 10:46:18.276 4778 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:18.279 4778 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:18.280 4778 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525 on network daa3e87e-298e-4b6d-813d-5be8d62b3525 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-13 10:46:18.281 4778 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525 on network daa3e87e-298e-4b6d-813d-5be8d62b3525 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-13 10:46:22.399 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.425 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.447 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.474 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.493 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.524 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap842131e0-ab', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.565 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-o', 'link', 'show', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.591 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-13 10:46:22.621 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap842131e0-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:22.623 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-13 10:46:22.626 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap842131e0-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:22.628 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap842131e0-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:22.629 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'842131e0-aba3-4679-a961-3a8b0f3a05cb', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:72:9f:cf'})), record=tap842131e0-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:22.663 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap842131e0-ab', 'address', 'fa:16:3e:72:9f:cf'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.682 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.701 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap842131e0-ab', 'netns', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.735 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap842131e0-ab', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.757 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'link', 'set', 'tap842131e0-ab', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.779 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Acquired semaphore "iptables-qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:22.781 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.809 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.839 4778 DEBUG neutron.agent.linux.iptables_manager [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-13 10:46:22.840 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Releasing semaphore "iptables-qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:22.843 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-o', 'link', 'show', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.862 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', 'addr', 'show', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.880 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'addr', 'add', '11.141.1.2/24', 'scope', 'global', 'dev', 'tap842131e0-ab', 'brd', '11.141.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.900 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.916 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Setting IPv4 gateway for dhcp netns on net daa3e87e-298e-4b6d-813d-5be8d62b3525 to 11.141.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-13 10:46:22.918 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.141.1.1', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.935 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.952 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Cleaning stale devices for network daa3e87e-298e-4b6d-813d-5be8d62b3525 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-13 10:46:22.953 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.968 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:22.986 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Building initial lease file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-13 10:46:22.988 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Done building initial lease file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases with contents:
1536936382 fa:16:3e:72:9f:cf 11.141.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-13 10:46:22.989 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:46:22.991 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:46:22.995 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:22.997 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host', '--addn-hosts=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap842131e0-ab', '--dhcp-range=set:tag0,11.141.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.042 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.043 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.045 4778 DEBUG neutron.agent.linux.external_process [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:46:23.047 4778 INFO neutron.agent.dhcp.agent [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Finished network daa3e87e-298e-4b6d-813d-5be8d62b3525 dhcp configuration
2018-09-13 10:46:23.048 4778 INFO neutron.agent.dhcp.agent [req-8a2cf9fc-f5c9-45c2-9719-e9d9e3ca0877 - - - - -] Synchronizing state complete
2018-09-13 10:46:23.050 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:23.051 4778 INFO neutron.agent.dhcp.agent [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-13T14:46:18Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.2'}], id=842131e0-aba3-4679-a961-3a8b0f3a05cb, mac_address=fa:16:3e:72:9f:cf, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:46:20Z
2018-09-13 10:46:23.053 4778 DEBUG neutron.agent.dhcp.agent [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:23.056 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:46:23.059 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:46:23.068 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14824'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.090 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:46:23.092 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.113 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.132 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.133 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.134 4778 DEBUG neutron.agent.linux.external_process [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:46:23.135 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:23.137 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:23.137 4778 INFO neutron.agent.dhcp.agent [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-13T14:46:18Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-daa3e87e-298e-4b6d-813d-5be8d62b3525, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'ba58b8eb-e460-4169-bcf7-beadd61d992b', u'ip_address': u'11.141.1.2'}], id=842131e0-aba3-4679-a961-3a8b0f3a05cb, mac_address=fa:16:3e:72:9f:cf, name=, network_id=daa3e87e-298e-4b6d-813d-5be8d62b3525, port_security_enabled=False, project_id=d924abce208c4c66b8ca02355a868a67, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d924abce208c4c66b8ca02355a868a67, updated_at=2018-09-13T14:46:21Z
2018-09-13 10:46:23.139 4778 DEBUG neutron.agent.dhcp.agent [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:23.141 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Building host file: /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-13 10:46:23.143 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Done building host file /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-13 10:46:23.147 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14824'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.174 4778 DEBUG neutron.agent.linux.dhcp [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Reloading allocations for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-13 10:46:23.177 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-4', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.205 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525', 'ip', '-6', 'route', 'list', 'dev', 'tap842131e0-ab'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:23.229 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.230 4778 DEBUG neutron.agent.linux.utils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:23.231 4778 DEBUG neutron.agent.linux.external_process [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:46:23.233 4778 DEBUG oslo_concurrency.lockutils [req-9634f430-2f2c-4d08-8470-a3f9631c0ada - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:27.352 4778 DEBUG oslo_concurrency.lockutils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:27.354 4778 DEBUG neutron.agent.dhcp.agent [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:27.356 4778 DEBUG neutron.agent.linux.utils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command (rootwrap daemon): ['kill', '-9', '14824'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:27.373 4778 DEBUG neutron.agent.linux.utils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-13 10:46:27.417 4778 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap842131e0-ab) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-13 10:46:27.485 4778 DEBUG neutron.agent.linux.utils [req-f591b5ce-ebfb-4e3a-aef5-999ba92f8247 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:27.500 4778 DEBUG neutron.agent.linux.utils [req-f591b5ce-ebfb-4e3a-aef5-999ba92f8247 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:27.514 4778 DEBUG oslo_concurrency.lockutils [req-f591b5ce-ebfb-4e3a-aef5-999ba92f8247 - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:28.586 4778 DEBUG oslo_concurrency.lockutils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:29.037 4778 DEBUG neutron.agent.linux.utils [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:29.039 4778 DEBUG neutron.agent.linux.utils [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Unable to access /var/lib/neutron/external/pids/daa3e87e-298e-4b6d-813d-5be8d62b3525.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:29.040 4778 DEBUG neutron.agent.linux.external_process [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:46:29.041 4778 DEBUG neutron.agent.dhcp.agent [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Calling driver for network: daa3e87e-298e-4b6d-813d-5be8d62b3525 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-13 10:46:29.044 4778 DEBUG neutron.agent.linux.utils [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:29.045 4778 DEBUG neutron.agent.linux.utils [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-13 10:46:29.046 4778 DEBUG neutron.agent.linux.external_process [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] No process started for daa3e87e-298e-4b6d-813d-5be8d62b3525 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-13 10:46:29.047 4778 DEBUG neutron.agent.linux.dhcp [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] Unable to access /var/lib/neutron/dhcp/daa3e87e-298e-4b6d-813d-5be8d62b3525/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-13 10:46:29.048 4778 DEBUG neutron.agent.linux.dhcp [req-f0b7a26b-2886-48bc-8f54-0f5da8f55e46 - - - - -] No interface exists for network daa3e87e-298e-4b6d-813d-5be8d62b3525 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-13 10:46:29.080 4778 DEBUG neutron.agent.linux.utils [req-5239000a-110d-4beb-892d-39552fd020ac - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-13 10:46:29.098 4778 DEBUG neutron.agent.linux.dhcp [req-5239000a-110d-4beb-892d-39552fd020ac - - - - -] Namespace already deleted: qdhcp-daa3e87e-298e-4b6d-813d-5be8d62b3525 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-13 10:46:29.100 4778 DEBUG oslo_concurrency.lockutils [req-5239000a-110d-4beb-892d-39552fd020ac - - - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:30.973 4778 DEBUG oslo_concurrency.lockutils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Acquired semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-13 10:46:30.975 4778 DEBUG oslo_concurrency.lockutils [req-81870934-bb76-443e-a79c-54599f85be06 6bce0e5766c84e1a98c3cb5ccb2e3772 4cff21d35d8b43b9a2a1ce042bb4f9f9 - - -] Releasing semaphore "dhcp-agent-network-lock-daa3e87e-298e-4b6d-813d-5be8d62b3525" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-13 10:46:53.050 4778 DEBUG neutron.agent.dhcp.agent [req-8a2cf9fc-f5c9-45c2-9719-e9d9e3ca0877 - - - - -] resync (daa3e87e-298e-4b6d-813d-5be8d62b3525): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-13 10:46:53.052 4778 INFO neutron.agent.dhcp.agent [req-8a2cf9fc-f5c9-45c2-9719-e9d9e3ca0877 - - - - -] Synchronizing state
2018-09-13 10:46:54.064 4778 INFO neutron.agent.dhcp.agent [req-19252fe9-fa4e-4ebb-9c08-fa36e7b9f135 - - - - -] All active networks have been fetched through RPC.
2018-09-13 10:46:54.067 4778 INFO neutron.agent.dhcp.agent [-] Starting network bcf73fc2-8df6-4f12-801c-ef4abf269c73 dhcp configuration
2018-09-13 10:46:54.068 4778 INFO neutron.agent.dhcp.agent [-] Finished network bcf73fc2-8df6-4f12-801c-ef4abf269c73 dhcp configuration
2018-09-13 10:46:54.069 4778 INFO neutron.agent.dhcp.agent [req-19252fe9-fa4e-4ebb-9c08-fa36e7b9f135 - - - - -] Synchronizing state complete
2018-09-13 10:47:13.042 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-13 10:47:13.047 4778 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
